sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/queries/Query13.java

```diff
@@ -57,19 +57,8 @@ public Query13(NexmarkConfiguration configuration) {
     final Coder<Event> coder = events.getCoder();
     return events
         .apply("Pair with random key", ParDo.of(new AssignShardFn<>(configuration.numKeyBuckets)))
-        .apply(GroupByKey.create())
-        .apply(
-            "ExpandIterable",
-            ParDo.of(
-                new DoFn<KV<Integer, Iterable<Event>>, Event>() {
-                  @ProcessElement
-                  public void processElement(
-                      @Element KV<Integer, Iterable<Event>> element, OutputReceiver<Event> r) {
-                    for (Event value : element.getValue()) {
-                      r.output(value);
-                    }
-                  }
-                }))
+        .apply(Reshuffle.of())
```

Comment:
There was a purpose of using GBK not reshuffle.

Reshuffle is composed PTransform that uses non-standard windows, thus it lose the chance to hit some optimizations.

Reply:
Thanks, my goal is to test the reshuffle transform. Opened #16337 for adding a query 14. Closing this.